tests/core/gui/test_workspaces.py

```diff
@@ -30,7 +29,8 @@ async def test_add_workspace(
     monkeypatch.setattr(
         "parsec.core.gui.workspaces_widget.get_text_input", lambda *args, **kwargs: (workspace_name)
     )
-    await aqtbot.mouse_click(w_w.button_add_workspace, QtCore.Qt.LeftButton)
+    async with aqtbot.wait_signal(w_w.list_success, timeout=2000):
+        await aqtbot.mouse_click(w_w.button_add_workspace, QtCore.Qt.LeftButton)
```

Comment:
Why not use `aqtbot.wait_until`? @touilleMan and I noticed it to be more robust than `wait_signal`?

Reply:
I forgot that wait_until also as a timeout parameter. It's been fixed.